src/03_modules.py

```diff
@@ -10,22 +10,27 @@
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
+print("Command Line Arguments: ", sys.argv[0])
```

Comment:
This is confusing because there's only one arg when you run this, but it's looking for a loop through them like:
`for arg in sys.argv`
    `print(arg)`